lib/Sema/TypeCheckStorage.cpp

```diff
@@ -2191,6 +2191,7 @@ synthesizeAccessorBody(AbstractFunctionDecl *fn, void *) {
 
   switch (accessor->getAccessorKind()) {
   case AccessorKind::Get:
+  case AccessorKind::DistributedGet:
```

Comment:
This is a bit surprising, shouldn't this produce `getDistributedThunk().getBody()` or something like that?

Reply:
Hmmm, the _distributed_get is the thunk, so I think this is okey?